initgeneration/init.sh.tmpl

```diff
@@ -101,6 +101,25 @@ tenant ${host_tenant}
 isCloudNativeFullStack true" >> ${1}
 }
 
+writeConfMintTo() {
+  mkdir -p "${1}"
+  echo "k8s.pod.uid ${K8S_PODUID}
+k8s.pod.name ${K8S_PODNAME}
+k8s.namespace.name ${K8S_NAMESPACE}
+dt.kubernetes.workload.kind ${DT_WORKLOAD_KIND}
+dt.kubernetes.workload.name ${DT_WORKLOAD_NAME}
+dt.kubernetes.cluster.id ${cluster_id}" >> "${1}/${2}.properties"
+
+  echo "{
+\"k8s.pod.uid\": \"${K8S_PODUID}\",
+\"k8s.pod.name\": \"${K8S_PODNAME}\",
+\"k8s.namespace.name\": \"${K8S_NAMESPACE}\",
+\"dt.kubernetes.workload.kind\": \"${DT_WORKLOAD_KIND}\",
+\"dt.kubernetes.workload.name\": \"${DT_WORKLOAD_NAME}\",
+\"dt.kubernetes.cluster.id\": \"${cluster_id}\"
+}" >> "${1}/${2}.json"
```

Comment:
would it be possible to use `jq` here or is that not available on the image?

Reply:
jq command is not available